workers/unity/Packages/com.improbable.gdk.core/Utility/EntityBuilder.cs

```diff
@@ -61,6 +85,14 @@ public EntityBuilder AddPosition(double x, double y, double z, string writeAcces
             return AddComponent(new ComponentData(schemaData), writeAccess);
         }
 
+        /// <summary>
+        ///     Sets the Persistence component to the entity template.
+        /// </summary>
+        /// <remarks>
+        ///    Persistence denotes whether the entity will be included in snapshots.
+        /// </remarks>
+        /// <param name="persistence">The value of persistence.</param>
+        /// <returns>Itself.</returns>
```

Comment:
You added a `.` to "Itself" here, should all of them have dots or none?

Reply:
This one is particularly terse